packages/mobile/src/screens/favorites-screen/EmptyTab.tsx

```diff
@@ -0,0 +1,82 @@
+import { ReactNode, useCallback } from 'react'
+
+import { Image, Text, View } from 'react-native'
+
+import Sophisticated from 'app/assets/images/emojis/face-with-monocle.png'
+import Button, { ButtonType } from 'app/components/button'
+import { EmptyCard } from 'app/components/core'
+import { useNavigation } from 'app/hooks/useNavigation'
+import { font, makeStyles } from 'app/styles'
+
+const messages = {
+  afterSaved: "Once you have, this is where you'll find them!",
+  goToTrending: 'Go to Trending'
+}
+
+type EmptyTabProps = {
+  message: string | ReactNode
+}
+
+const useStyles = makeStyles(({ palette, spacing }) => ({
+  tabContainer: {
+    backgroundColor: palette.white,
+    borderRadius: 6,
+    paddingVertical: spacing(8),
+    paddingHorizontal: spacing(6)
+  },
+  tabText: {
+    ...font('medium'),
+    fontSize: 16,
+    lineHeight: 26,
+    letterSpacing: 0.2,
+    marginBottom: spacing(4),
+    color: palette.neutral,
+    textAlign: 'center'
+  }
+}))
+
+export const EmptyTab = ({ message }: EmptyTabProps) => {
```

Comment:
This is great! we have a basic impl of this in the profile page, so ill def use this one instead cause its more built out